src/core/src/core_logic/PatchInstaller.py

```diff
@@ -159,6 +159,12 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
         self.telemetry_writer.write_event("Security packages out of the final package list: " + str(sec_packages), Constants.TelemetryEventLevel.Verbose)
         self.status_handler.set_package_install_status_classification(sec_packages, sec_package_versions, classification="Security")
 
+        if package_manager.get_package_manager_setting(Constants.PKG_MGR_SETTING_IDENTITY) == Constants.APT:
```

Comment:
Non-blocking: Review if this can be fully encapsulated within the package manager class.

Reply:
Added a method to package_mananger for the same.